src/Service/TraceContext/TraceContextService.php

```diff
@@ -76,6 +76,10 @@ public function handleResponse(Response $response, TraceContext $context): void
 
     public function handleClientRequest(TraceContext $trace, string $method, string $url, array $options = []): array
     {
+        if ($trace->getTraceId() === null) {
```

Comment:
This check is similar to the check in the method above, and the check in monolog/sentry handler